### PR TITLE
Assi4

### PR DESCRIPTION
**Pull request by Formil**
Solucionada anterior pull request con los archivos correctos

### CODE REVIEW

File: Assignment4/Formil-b190422/task07.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+from rdflib.plugins.sparql import prepareQuery
+
+# Define the SPARQL query
+query_str = """
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?subClass
+WHERE {
+  ?subClass rdfs:subClassOf <http://somewhere#LivingThing>.
```
Comment (anaigmo) [inline Assignment4/Formil-b190422/task07.py:35]: and subclasses of subclasses?

File: Assignment4/Formil-b190422/task07.py
Comment (anaigmo) [inline Assignment4/Formil-b190422/task07.py:1]: Missing rdflib alternatives for task 7.1-3

File: Assignment4/Formil-b190422/task07.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+"""**TASK 7.4:  List the name of the persons who know Rocky**"""
+
+query_str = """
+PREFIX vcard: <http://www.w3.org/2001/vcard-rdf/3.0#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+
+SELECT ?name
+WHERE {
+  ?person <http://www.w3.org/2001/vcard-rdf/3.0/FN> ?name.
+  ?person <http://xmlns.com/foaf/0.1/knows> <http://somewhere#RockySmith>.
```
Comment (anaigmo) [inline Assignment4/Formil-b190422/task07.py:93]: it's easier if you use prefixes :)

**Comment by dgarijo** [comment]
If you have another incorrect PR open, please close it

**Comment by Formil** [comment]
> If you have another incorrect PR open, please close it

I already sent an email to you, but sorry for the confusion again, I've already closed the other one pull request with the incorrect year's exercises, I will continue to edit this with the corrections indicated by Ana.

**Comment by dgarijo** [comment]
ok!

**Comment by Formil** [comment]
Added the task7 rdflib implementations that were missing

**Comment by anaigmo** [comment]
Submitted and fixed, no more PRs allowed

**Pull request merged**